components/news/Home.vue

```diff
@@ -0,0 +1,115 @@
+<template>
+  <div class="w-full">
+    <nuxt-link :to="'/news/' + news.slug">
+      <div class="news-image">
+        <img :src="newsImage" :alt="news.title.rendered" class="">
+      </div>
+    </nuxt-link>
+    <div class="grid news-infos">
+      <div class="news-date">
+        <div class="newsgeneral">
+          <nuxt-link to="news/">
+            <h3 class="m-0">
+              News
+            </h3>
+          </nuxt-link>
+        </div>
+        <div class="newsdate">
+          <p>{{ format(new Date(news.modified), 'MM') }}</p>
```

Comment:
It seems like we are using time and date in many places. 

When you develop in js / vue what's preferred? To always just format these and create time / date objects inplace or to have generic methods somewhere and use those for all over the application? 

Sorry if this is dumb question but I don't know what the best practices are for current front-end development.

Reply:
format is a part of date-fns library for showing the time in different format https://date-fns.org/v2.22.1/docs/format
new Date is a js Date object